indico/modules/events/settings.py

```diff
@@ -259,3 +259,8 @@ def get_themes_for(self, event_type):
 }, acls={
     'authorized_creators'
 })
+
+event_language_settings = EventSettingsProxy('language', {
```

Comment:
Maybe move this up a bit so all the EventSettingsProxy objects are next to each other (`unlisted_events_settings` is a normal SettingsProxy)